src/main/scala/wdl4s/expression/ValueEvaluator.scala

```diff
@@ -113,7 +113,30 @@ case class ValueEvaluator(override val lookup: String => WdlValue, override val
               case o: WdlObjectLike =>
                 o.value.get(rhs.getSourceString) match {
                   case Some(v:WdlValue) => Success(v)
-                  case None => Failure(new WdlExpressionException(s"Could not find key ${rhs.getSourceString}"))
+                  case None =>
```

Comment:
Yes it is - it's just this whole file is pretty much a giant match case and it seemed even more confusing to me to make a special case method for this and not for the rest, and I didn't want to refactor the entire thing..